### DISCUSSION TITLE
How can I solve this error?

### DISCUSSION THREAD

**Discussion by waterproofneighbor** [andrey-yantsen/tlscontact-appointment-finder#8] [Q&A]
Got an exception! HTTPConnectionPool(host='selenium', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff6c2dccbe0>: Failed to establish a new connection: [Errno 111] Connection refused'))

**Comment by worried-networking** [comment]
Hi!

Most likely, it's caused because of the bad docker-compose config, sorry :) Sometimes Selenium loads slower than the second container.

As a "solution", try running the Selenium container manually and then waiting for a few seconds before starting the second one.
